cmd/web-handlers.go

```diff
@@ -21,6 +21,11 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/dustin/go-humanize"
```

Comment:
Then perhaps sublime is formatting it wrong, they should avoid that.  You should use `goimports` which would properly format the files.